### PR TITLE
i want to merge feature-1 branch to main branch

### PR DESCRIPTION
**Pull request by mohammedabdulah**
i have added this and this
now i want to merge this branch into main branch
please accept pull request

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -9,3 +9,7 @@ now it is time for commit-2
 
 ### third commit
 now i am commiting for thired time
+
+### fourth commit
```
Comment (mohammedabdulahquraishi) [inline README.MD:13]: is this line required here

Comment (mohammedabdulah) [reply README.MD:13]: yes sir

File: README.MD [hunk 2]
```diff
@@ -9,3 +9,7 @@ now it is time for commit-2
...
+now i am commiting again for fourth time
+
```
Comment (mohammedabdulahquraishi) [inline README.MD:15]: add something extra here

Comment (mohammedabdulah) [reply README.MD:15]: added sir

**Review by mohammedabdulahquraishi — CHANGES_REQUESTED** [review]
ok make these changes

**Comment by mohammedabdulah** [comment]
dfgjhn

**Review by mohammedabdulahquraishi — APPROVED** [review]

**Comment by mohammedabdulah** [comment]
thanks

**Pull request merged**